src/network/network.cpp

```diff
@@ -116,25 +116,25 @@ Json::Value Network::getStatus() { return taraxa_capability_->getStatus(); }
 std::vector<NodeID> Network::getAllPeersIDs() const { return taraxa_capability_->getAllPeersIDs(); }
 
 void Network::onNewBlockVerified(shared_ptr<DagBlock> const &blk) {
-  tp_.post([=] {
+  tp_.post([=, this] {
```

Comment:
So maybe a better way to do it is capture all needed stuff explicitly? Like you did with `restartSyncingPbft`
Dont want to add the same comment for every line, so thing to notice: if you will be changing it there are some other lambdas also need this changes. In this file, taraxa_capability.cpp, thread_pool.cpp